website/docs/components/pagination/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,44 @@
+## Usage
+
+### When to use
+
+- To break down large content into pages. Usually paired with tables, but works with other types of page content.
+
+### When not to use
+
+- As a navigation control for a flow or to pair with a stepper, i.e. for a guide, tutorial, setup flow.
+- As a controller to switch between multiple views. Use [Tabs](/components/tabs/) instead.
+
+### Numbered vs Compact
+
+In most cases, the numbered pagination provides a greater user experience. It allows users to jump between pages and always return to the first page or go to the last page without navigating through the pages manually.
+
+If your product only has cursor pagination, it won't be able to support the numbered variant. Only applications with offset pagination can implement the numbered pagination variant.
+
+![Numbered pagination example](/assets/components/pagination/pagination-offset-example.png =50%x50%)
+- Supported by offset (page-based) pagination.
+
+![Compact pagination example](/assets/components/pagination/pagination-cursor-example.png =25%x25%)
+- Supported by offset and cursor based pagination.
+
+!!! Warning 
+
+We strongly recommend talking to your engineering team before using a specific variant. Learn more about [offset and cursor pagination] (https://ignaciochiazzo.medium.com/paginating-requests-in-apis-d4883d4c1c4c)
+
+!!!
```

Comment:
It feels like we're trying to do too much in one section (i.e., cursor vs. offset and numbered vs. compact). Have you thought about how to provide more helpful information about cursor vs. offset pagination directly in the docs (vs. just linking out to an article)? Admittedly I had no idea what cursor pagination was until you told us about it in a design review.

Reply:
Thanks for reviewing. I'm addressing the other changes and will tackle this one in a separate commit as I need to think a bit about it.